vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/gemini/post_account/post_account_gemini.cc

```diff
@@ -0,0 +1,152 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/endpoint/gemini/post_account/post_account_gemini.h"
+
+#include <utility>
+
+#include "base/json/json_reader.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ledger/internal/endpoint/gemini/gemini_utils.h"
+#include "bat/ledger/internal/ledger_impl.h"
+#include "net/http/http_status_code.h"
+
+using std::placeholders::_1;
+
+namespace ledger {
+namespace endpoint {
+namespace gemini {
+
+PostAccount::PostAccount(LedgerImpl* ledger) : ledger_(ledger) {
+  DCHECK(ledger_);
+}
+
+PostAccount::~PostAccount() = default;
+
+std::string PostAccount::GetUrl() {
+  return GetApiServerUrl("/v1/account");
+}
+
+type::Result PostAccount::CheckStatusCode(const int status_code) {
```

Comment:
For this function `HTTP_NOT_FOUND` returns `EXPIRED_TOKEN` but `CheckStatusCode` in `gemini_utils.cc` returns `NOT_FOUND`. Returning `EXPIRED_TOKEN` for `HTTP_FORBIDDEN` is confusing

Reply:
resolved, using `CheckStatusCode` in `gemini_utils.cc`